### PR TITLE
Update toml configs for identity xml

### PR DESCRIPTION
**Pull request by janakamarasena**

### CODE REVIEW

File: en/docs/learn/basic-client-profile-with-playground.md [hunk 1]
```diff
@@ -405,14 +395,13 @@ scenario.
...
-            follows:
+            Open the `              deployment.toml             ` file found in the
+            `              <IS_HOME>/repository/conf             `
+            directory and change the following configuraiton
         
-            ``` xml
-                <TokenPersistenceProcessor>org.wso2.carbon.identity.oauth.tokenprocessor.EncryptionDecryptionPersistenceProcessor</TokenPersistenceProcessor>
+            ``` toml
+                [oauth]
+                token_storage_method="encrypted"
```
Comment (YvonneW) [inline en/docs/learn/basic-client-profile-with-playground.md]: Fix indentation

File: en/docs/learn/invoke-the-oauth-introspection-endpoint.md [hunk 2]
```diff
@@ -66,15 +66,12 @@ invoke the OAuth introspection endpoint for the super tenant users.
...
-        `           <IS_HOME>/repository/conf/identity/identity.xml          `
+        the permissions can be added to the `<IS_HOME>/repository/conf/deployment.toml`         `
         file.
     
         ``` toml
-            <Resource context="(.*)/oauth2/introspect(.*)" secured="true" http-method="all">
-                <Permissions>/permission/admin/manage/identity/applicationmgt/view</Permissions>
-            </Resource>
+            [resource_access_control.introspect]
+            permissions = ["/permission/admin/manage/identity/applicationmgt/view","/permission/admin/login"]
```
Comment (YvonneW) [inline en/docs/learn/invoke-the-oauth-introspection-endpoint.md]: Fix indentation

File: en/docs/learn/basic-client-profile-with-playground.md (hunk 1 above)
Comment (janakamarasena) [reply en/docs/learn/basic-client-profile-with-playground.md]: fixed

File: en/docs/learn/invoke-the-oauth-introspection-endpoint.md (hunk 2 above)
Comment (janakamarasena) [reply en/docs/learn/invoke-the-oauth-introspection-endpoint.md]: fixed

**Pull request merged**